src/portal/views/Account.vue

```diff
@@ -182,17 +175,21 @@ export default class AccountView extends Vue {
             this.loadingTwinCreate = false;
           } else {
             // Loop through Vec<EventRecord> to display all events
-            events.forEach(({ phase, event: { data, method, section } }) => {
+            events.forEach(async ({ phase, event: { data, method, section } }) => {
```

Comment:
I think it would be better if we used a normal for of loop with async.

Reply:
updated!